### ISSUE TITLE
CircleCI integration

### ISSUE THREAD

**Issue by lemmatum** [PlasmaPy/SpectroscoPyx#25]
Implement CircleCI for continuous integration, similar to PlasmaPy.

**Comment by lemmatum** [comment]
Make sure test-html and test-latex are separate for speedier testing.

**Comment by lemmatum** [comment]
@namurphy @StanczakDominik I'm currently getting this error for circleci:
```
Build-agent version 0.0.4869-fac853b (2018-04-17T20:59:55+0000)
Starting container spectroscopyx/documentation-builder:latest
  image cache not found on this host, downloading spectroscopyx/documentation-builder:latest
Error response from daemon: repository spectroscopyx/documentation-builder not found: does not exist or no pull access
```

I think I need someone with admin access to setup the documentation-builder container for spectroscopyx.

**Comment by lemmatum** [comment]
ping @StanczakDominik

**Comment by StanczakDominik** [comment]
I've got it sorta working, but I'm gonna try another approach and do what Cadair did at PlasmaPy 0.1 release and retemplate this.  I think that's gonna prove more helpful in the long run.

> Make sure test-html and test-latex are separate for speedier testing.

I cut that out a while ago as it's better to run the Docker image once and run HTML first, LaTeX later.

**Comment by lemmatum** [comment]
@StanczakDominik it's further along than what I could do! Did this involve administrative controls on CircleCI?

> do what Cadair did at PlasmaPy 0.1 release and retemplate this.
 Does this refer to the template used for the documentation as CircleCI is currently failing to build LaTeX?

> I cut that out a while ago as it's better to run the Docker image once and run HTML first, LaTeX later.
Optimizations! :scream:

**Comment by StanczakDominik** [comment]
No, actually, I did nothing admin-based to CircleCi. You were simply referring to a Docker image that doesn't exist. :D I turned that around to refer to the one I made earlier for PlasmaPy, it works all right. The CircleCI LaTeX failure is a bunch of requirements that we don't actually have in there right now - sphinx-astropy, a few others I'm misremembering. Retemplating this should solve it, but it's a whole bunch of work and I'm feeling kinda sleepy (didn't sleep well at all last night), so I'll just finish it up tomorrow, I think.
